cg/meta/workflow/nf_analysis.py

```diff
@@ -103,3 +106,72 @@ def get_metrics_deliverables_path(self, case_id: str) -> Path:
         return Path(self.root_dir, case_id, f"{case_id}_metrics_deliverables").with_suffix(
             FileExtensions.YAML
         )
+
```

Comment:
We'll need to cleanup this method at some point since it's not very readable. But let's do it after refactoring the nextflow and tower classes